src/fleet/azext_fleet/_params.py

```diff
@@ -0,0 +1,37 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+import os
+from argcomplete.completers import FilesCompleter
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    file_type
+)
+from azext_fleet._validators import validate_member_cluster_id
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('fleet') as c:
+        c.argument('name', options_list=['--name', '-n'], help='Specify the fleet name.')
+
+    with self.argument_context('fleet create') as c:
+        c.argument('tags', tags_type)
```

Comment:
I suppose `tags` is supported by CLI framework by default

Reply:
unsure, borrowed from aks-preview
https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/_params.py#L186